doc/man7/openssl-glossary.pod

```diff
@@ -30,11 +62,73 @@ It is defined in ITU-T document X.690:
 
 L<https://www.itu.int/rec/T-REC-X.690>
 
+=item Encoder
+
+An encoder is a type of algorithm used for encoding keys and parameters to some
+external format such as PEM or DER.
+
+L<OSSL_ENCODER_CTX_new_for_pkey(3)>
+
+=item Fetching
+
+Fetching is the process of looking through the available algorithm
+implementations, applying selection criteria (via a property query string), and
+finally choosing the implementation that will be used. Fetches can either be
+explicit where a function call is made to obtain an algorithm object
```

Comment:
Could this be two items here for implicit and explicit

Reply:
I've now got different sections for "Fetching", "Explicit Fetching" and "Implicit Fetching"